easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2020b.eb

```diff
@@ -0,0 +1,69 @@
+##
+# This file is an EasyBuild recipy as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# https://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Updated to use gompi-2020b toolchain:
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+source_urls = ['https://github.com/soedinglab/hh-suite/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd67f7f3bf601e48c9c0bc4cf1fbe3b946f787a808bde765e9436a48d27b0964']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Perl', '5.32.0'),
+]
+
+_binaries = [
+    'hhalign', 'hhalign_mpi', 'hhalign_omp',
+    'hhblits', 'hhblits_mpi', 'hhblits_omp',
+    'hhsearch', 'hhsearch_mpi', 'hhsearch_omp',
+    'hhmake', 'hhfilter', 'hhconsensus',
+]
+
+_scriptfiles = [
+    'a3m.py', 'check_a3m.py', 'cif2fasta.py', 'ffindex.py', 'get_a3m_size.py',
+    'hhmakemodel.py', 'hh_reader.py', 'hhsuitedb.py', 'cif2fasta.py',
```

Comment:
Checking for all of these is overkill imho, just a couple of fine.

Reply:
Fair enough. I thought I am very thorough here and check for all of them, rather than a few. 
I basically removed the first line as the 2nd one contains entries which are all somehow start with 'hh' apart from the last one.